site/content/integrations/grafana.md

```diff
@@ -3,19 +3,60 @@ title: "Grafana"
 weight: 3
 ---
 
+## Overview 
 
-M3 supports a variety of Grafana integrations.
+Grafana is an open-source visualization tool used to dashboarding, querying, and monitoring databases, especially used for time series databases. See more [here](https://grafana.com/). 
```

Comment:
```suggestion
[Grafana](https://grafana.com/) is an open-source visualization tool used for dashboarding, querying, and monitoring data sources, especially time series databases. 
```

I think Grafana can do more than monitor databases, so added a bit more, and also avoid "here" links as they tell people nothing about what they're clicking.